cirq/protocols/approximate_equality_protocol.py

```diff
@@ -94,6 +95,11 @@ def approx_eq(val: Any, other: Any, *, atol: Union[int, float] = 1e-8) -> bool:
     if isinstance(val, str):
         return val == other
 
+    if isinstance(val, sympy.core.mul.Mul) or isinstance(
+            other, sympy.core.mul.Mul):
+        delta = sympy.simplify(sympy.Abs(other - val))
```

Comment:
Why do we need the simplify here?

Reply:
If I don't do that, the unit test fails. Upon inspection, if I print `delta`, I see `delta=Abs((s + 1.51)/pi - (s + 1.515)/pi)`

Reply:
Got it - however, this is still not kosher. There might be a case where you can't evaluate to a number literal and then this question is undecidable. We should handle that and throw an error instead of silently returning False.

```sympy.Abs(val - other).simplify().is_number``` should give you a good indication if the simplification worked.

Please add a test case covering that too.